packages/lit-ssr/package.json

```diff
@@ -58,6 +58,7 @@
     "koa": "^2.7.0",
     "koa-node-resolve": "^1.0.0-pre.5",
     "koa-static": "^5.0.0",
+    "lit": "^2.0.0-pre.1",
     "lit-element": "^3.0.0-pre.2",
     "lit-html": "^2.0.0-pre.4",
```

Comment:
Note, the only reason lit-ssr now still relies on `lit-element` and `lit-html` are to import their respective `private-ssr-support.js` packages, since we don't forward those out through `lit`. That seems ... ok?

Reply:
yeah